src/component/framework/bootstrap/association.html

```diff
@@ -1,20 +1,19 @@
 <template>
   <require from="./../../../attributes"></require>
   <require from="./form-group"></require>
-  <form-group role="group" element.bind="formField.element" message.bind="formField.message" description.bind="formField.description">
+  <form-group errors.bind="errors" role="group" element.bind="formField.element" message.bind="formField.message" description.bind="formField.description">
     <association-select
       attributes.bind        = "formField.element.attributes"
       multiple.bind          = "formField.element.multiple"
       value.bind             = "formField.value"
       property.bind          = "formField.element.property"
       repository.bind        = "formField.element.repository"
       resource.bind          = "formField.element.resource"
-
+      type.bind              = "formField.element.selectType"
```

Comment:
This allows for checkboxes or select type to be chose for aurelia-orm's association-select component.